sql/mob_pools.sql

```diff
@@ -6078,7 +6078,7 @@ INSERT INTO `mob_pools` VALUES (6016,'ingrid_ii','Ingrid',149,0x00001E0C00000000
 INSERT INTO `mob_pools` VALUES (6017,'arciela_ii','Arciela',149,0x00000D0C00000000000000000000000000000000,5,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,426,1,0,1132);
 INSERT INTO `mob_pools` VALUES (6018,'iroha_ii','Iroha',149,0x0000280C00000000000000000000000000000000,12,3,3,240,100,0,0,0,0,0,0,32,0,3,0,0,427,1,0,1133);
 INSERT INTO `mob_pools` VALUES (6019,'shantotto_ii','Shantotto',153,0x0000260C00000000000000000000000000000000,4,0,2,240,100,0,2,0,0,0,0,32,0,3,0,0,428,1,0,1134);
--- Reserved for future Trust 6020
+INSERT INTO `mob_pools` VALUES (6020,'Luopan','Luopan',503,0x0000280B00000000000000000000000000000000,1,1,2,360,100,0,2,0,0,0,0,8,0,0,0,0,0,16,0,755);
```

Comment:
reserved for a reason do not steal the ID!

Reply:
Ah ok, will change this.

Reply:
Done it is now in a nice gap, and set as 6040 ✔